docs/persistence/elastic-search.md

```diff
@@ -0,0 +1,20 @@
+# Elastic Search
+
+!!! Info
+    This feature is only available in GCP clusters.
+
+The NAIS platform offer Elastic Search via Aiven.
+
+## Get your own
+As there are few teams that need an Elastic Search instance we use a IaC-repo to provision each instance.
+Head over to [aiven-iac](https://github.com/navikt/aiven-iac#elastic-search) to learn how to get your own instance.
+
+### Username and password
+For now we are manually distributing the username and password for each instances.
+This user will be an admin user, which use can use to create the specific read/write users that you need.
```

Comment:
`This user will be an admin user which in turn can be used to create read/write users.` perhaps? at least `..., which can be used to create ...`